index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portforio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/styles.css">
+</head>
+<body>
+ <header>
+    <a href="#home"> <img src="images/My-Logo.png"> </a>  
+    <nav> 
+        <img src="images/Union.svg">
+    </nav>
+ </header>
+      <div class="container">
+         <section>
+            <div id="text1">
+            <h1>I’m Eddy Glad to see you!</h1>
+        </div>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me</p>
+             <h4>LET’S CONNECT</h4>
+             <ul>
+                 <li> <a href='https://twitter.com/eddy56388406'> <img src='/images/Vector.png'> </a> </li>
+                 <li> <a href='https://www.linkedin.com/in/pray-eddy-shimwa-18a6751b6/'> <img src='/images/Linkedin icon.png'> </a> </li>
+                 <li> <a href='https://github.com/EddyShimwa'> <img src='/images/Vector (1).png'> </a> </li>
+                 <li> <a href='N/A'> <img src='/images/Vector (2).png'> </a> </li>
+                 <li> <a href='N/A'> <img src='/images/Vector (3).png'> </a> </li>
```

Comment:
Kindly fix the paths to this icons. remove the `/` before them
your path should read `<li> <a href='N/A'> <img src='images/Vector (2).png'> </a> </li>`
They currently are unrendered. in the browser:
<img width="699" alt="Screenshot 2022-10-28 at 11 19 35" src="https://user-images.githubusercontent.com/48546992/198541546-f986271b-b072-4bf8-9a1c-472bba5c2ea6.png">